source/server-discovery-and-monitoring/tests/integration/closeInUse-pool-clear.yml

```diff
@@ -0,0 +1,108 @@
+# Test connection pool clear(closeInUse:true).
+runOn:
+    # This test requires failCommand appName, and
+    # failCommand blockConnection with closeConnection:true (SERVER-53512).
+    - minServerVersion: "4.9"
+
+database_name: &database_name "sdam-tests"
+collection_name: &collection_name "closeInUse"
+
+data: []
+
+tests:
+  - description: pool clear close in use true generates expected events
+    useMultipleMongoses: false,
+    clientOptions:
+      connectTimeoutMS: 500, 
+      heartbeatFrequencyMS: 500
+      appname: "closeInUse"
+      retryReads: false
+      minPoolSize: 0
+    operations:
+      - name: insertMany
+        object: collection
+        arguments:
+          documents:
+            - _id: 1
+      - name: startThread
+        object: testRunner
+        arguments:
+          name: thread1
+      # simulate a long-running query
+      - name: runOnThread
```

Comment:
When this is ported to the unified format, we'll want to add an `expectError` to this operation and a corresponding `waitForThread` operation to ensure it did indeed fail.

Reply:
done